### PR TITLE
Allow custom class definition for text item

### PR DESCRIPTION
**Pull request by cool-runnings**
Added class attribute to text item.
Text creation is now done with createDefaultWidget

### CODE REVIEW

**Comment by ChristianMayer** [comment]
Generally fine - but please clean up the code, i.e. delete and not only comment the old code

**Comment by joltcoke** [comment]
Please adjust the PR title to be: "Allow custom class definition for text item" or something a bit more expressive.

Label: Enhancement
Milestone: 1.0.0

**Comment by ChristianMayer** [comment]
Apart from the code cleanup this pull request has unwanted side effects.

Without pull request in the demo config:
text1

With pull request in the demo config:
text2

When both issues are fixed I can pull the request.

File: src/structure/pure/_common.js [hunk 1]
```diff
@@ -401,7 +401,7 @@ function VisuDesign() {
     if( $element.attr('flavour') ) flavour = $element.attr('flavour');// sub design choice
     if( flavour ) classes += ' flavour_' + flavour;
     if($element.attr('class')) classes += ' custom_' + $element.attr('class');
-    var label = this.extractLabel( $element.find('label')[0], flavour );
+    var label = (widgetType==='text')?this.extractLabel( $element.find('label')[0], flavour, '' ):this.extractLabel( $element.find('label')[0], flavour );
```
Comment (ChristianMayer) [inline src/structure/pure/_common.js]: I'm not too happy about special casing a generic function (anything in _common.js is generic). A better solution is usually a generalisation.

But I'll pull anyway, we can fix that later.

**Pull request merged**